src/integrationtests/java/com/aws/greengrass/integrationtests/BaseITCase.java

```diff
@@ -148,4 +150,14 @@ public static void deleteWindowsTestUser(String username) throws IOException, In
         // Delete test user's home directory
         FileUtils.deleteQuietly(new File("C:\\Users\\" + username));
     }
+
+    @SuppressWarnings({"PMD.SignatureDeclareThrowsException"})
+    public static void overrideInstanceField(Object target, String field, Object value) throws Exception {
+        Field f = target.getClass().getDeclaredField(field);
+        f.setAccessible(true);
+        Field modifiersField = Field.class.getDeclaredField("modifiers");
+        modifiersField.setAccessible(true);
+        modifiersField.setInt(f, f.getModifiers() & ~Modifier.FINAL);
+        f.set(target, value);
```

Comment:
Why do you need reflection?

Reply:
this is old change and no longer part of the commit, not sure how you are seeing it.